src/components/Search.js

```diff
@@ -0,0 +1,89 @@
+import React, { Fragment, useState, useEffect } from 'react';
+import { Global, css } from '@emotion/core';
+import { DocSearch } from '@docsearch/react';
+
+const Search = () => (
+  <Fragment>
+    <Global
+      styles={css`
+        :root {
+          --docsearch-primary-color: #ff6b81;
+          --docsearch-text-color: rgb(245, 246, 247);
+          --docsearch-container-background: rgba(9, 10, 17, 0.8);
+          --docsearch-modal-background: #281e36;
+          --docsearch-modal-shadow: inset 1px 1px 0 0 rgb(44, 46, 64),
+            0 3px 8px 0 rgb(0, 3, 9);
+          --docsearch-searchbox-background: rgb(9, 10, 17);
+          --docsearch-searchbox-focus-background: #000;
+          --docsearch-hit-color: rgb(190, 195, 201);
+          --docsearch-hit-shadow: none;
+          --docsearch-hit-background: rgb(9, 10, 17);
+          --docsearch-key-gradient: linear-gradient(
+            -26.5deg,
+            rgb(114 86 111) 0%,
+            rgb(65 46 80) 100%
+          );
+          --docsearch-key-shadow: inset 0 -2px 0 0 rgb(81 31 82),
+            inset 0 0 1px 1px rgb(125 81 111), 0 2px 2px 0 rgba(3, 4, 9, 0.3);
+          --docsearch-footer-background: #2f263c;
+          --docsearch-footer-shadow: inset 0 1px 0 0 rgba(73, 76, 106, 0.5),
+            0 -4px 8px 0 rgba(0, 0, 0, 0.2);
+          --docsearch-logo-color: #fff;
+          --docsearch-muted-color: rgb(127, 132, 151);
+          --docsearch-key-shadow: inset 0 -2px 0 0 rgb(40, 45, 85),
+            inset 0 0 1px 1px rgb(81, 87, 125), 0 2px 2px 0 rgba(3, 4, 9, 0.3);
+          --docsearch-footer-background: #2f263c;
+          --docsearch-footer-shadow: inset 0 1px 0 0 rgba(73, 76, 106, 0.5),
+            0 -4px 8px 0 rgba(0, 0, 0, 0.2);
+          --docsearch-logo-color: rgb(255, 255, 255);
+          --docsearch-muted-color: rgb(127, 132, 151);
+
+          .DocSearch-Button {
```

Comment:
This can be moved out of `:root` too.

Reply:
This can not be moved out of `:root` since the CSS variables are overridden for the search button to fit with the overall design.